lib/iris/tests/unit/analysis/test_PERCENTILE.py

```diff
@@ -293,6 +297,8 @@ def setUp(self):
         self.agg_method = PERCENTILE.lazy_aggregate
 
     def test_masked(self):
+        # Using (3,11) because np.percentile returns a masked array anyway with
+        # (2, 11)
         shape = (2, 11)
```

Comment:
Comment doesn't match `shape` here.  Since we're just checking that an exception is raised it probably doesn't matter whether it's (3, 11) or (2, 11).

Reply:
https://github.com/SciTools/iris/pull/4755/commits/7af47a14fce0bad6269c234ece83c43b5f0d359f